### PR TITLE
Print information about local plugins

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
When plugin in referenced via local path or github url, `appbuilder plugin` command
does not show information about it.
Read the package.json of the local plugin in order to get data for it.
Install the plugin referenced with github url to temp dir and read its package.json.

Fixes http://teampulse.telerik.com/view#item/303583

### CODE REVIEW

**Comment by teobugslayer** [comment]
```
does not prompt for Cordova upgrade to 3.7.0 when WPSdk is upgraded to 8.1 and FrameworkVersion is already 3.7.0 (550ms)
09:55:42     Init command tests
09:55:42       NativeScript project
09:55:42 AssertionError: NativeScript Blank template does not contain mandatory 'tns_modules' directory. This directory is required in init command. You should check if this is problem with the template or change init command to use another file.: expected false to be true
```

**Comment by rosen-vladimirov** [comment]
I'll fix the tests in separate PR and rebase this one after merging it :)

**Comment by rosen-vladimirov** [comment]
I've rebased this PR, so it should be fine now :)

File: lib/services/nativescript-project-plugins-service.ts [hunk 1]
```diff
@@ -358,6 +343,68 @@ export class NativeScriptProjectPluginsService implements IPluginsService {
...
+				pathToPlugin = pathToPlugin.replace("file:", "");
+			}
+
+			if(this.checkIsValidLocalPlugin(pathToPlugin).wait()) {
+				let fullPath = path.resolve(pathToPlugin);
+				let packageJsonContent = this.$fs.readText(path.join(fullPath, this.$projectConstants.PACKAGE_JSON_NAME)).wait();
+				plugin = this.constructNativeScriptPluginData(packageJsonContent).wait();
+			}
+
+			return plugin;
```
Comment (teobugslayer) [inline lib/services/nativescript-project-plugins-service.ts]: This returns implicitly `undefined`. Should we return an explicit `null` instead?

**Comment by teobugslayer** [comment]
Looks good, :+1: but check my remark.

**Pull request merged**